ledger/participant-integration-api/src/main/scala/platform/indexer/JdbcIndexer.scala

```diff
@@ -120,9 +116,9 @@ object JdbcIndexer {
       ledgerDao.initializeLedger(providedLedgerId)
     }
 
-    private def initOrCheckParticipantId(dao: LedgerDao)(
-        implicit executionContext: ExecutionContext,
-    ): Future[Unit] = {
+    private def initOrCheckParticipantId(
+        dao: LedgerDao,
+    )(implicit resourceContext: ResourceContext): Future[Unit] = {
```

Comment:
There's inconsistency in how you name your implicit resource contexts. I would recommend sticking to the naming you used here (`resourceContext`), since we also have other contexts we pass around.

Reply:
I tried to keep it as `resourceContext` except when it's the implicit parameter of a `ResourceOwner#acquire`, in which case it's `context`. Is this too confusing?